frontend/src/metabase/visualizations/components/settings/ColumnItem.jsx

```diff
@@ -30,7 +30,7 @@ const ColumnItem = ({
   draggable,
 }) => {
   return (
-    <ColumnItemRoot draggable={draggable} onClick={onClick}>
+    <ColumnItemRoot onClick={onClick} isDraggable={draggable}>
```

Comment:
Need to change the name of the `draggable` prop as it is an actual HTML prop on elements, and interfering with the `Sortable` HOC. Here it is only meant to change the cursor.